example/distancecast.js

```diff
@@ -274,7 +274,7 @@ function updateDistanceCheck( fastCheck ) {
 	targetContainer.updateMatrixWorld();
 	const targetToBvh =
 		new THREE.Matrix4()
-			.getInverse( terrain.matrixWorld )
+		    .copy( terrain.matrixWorld ).invert()
```

Comment:
Can we fix the indentation to be a tab here? I'll have fix the linting to make sure it runs on the example files, too 🤔

Reply:
hm i add example to lint but lint does not show this

Reply:
Thanks, I'll take a look!